<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MfaParametersForm</class>
 <widget class="QDialog" name="MfaParametersForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Análisis Multifractal - Parámetros</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources/resources.qrc">
    <normaloff>:/icons/mfa.png</normaloff>:/icons/mfa.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_10">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="instructionsLabel">
     <property name="styleSheet">
      <string notr="true">QLabel{background-color: rgb(255, 255, 255);}
</string>
     </property>
     <property name="text">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Parámetros de Entrada&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Seleccione las secuencias para las cuales realizará el análisis multifractal; a continuación especifique los valores &lt;span style=&quot; font-style:italic;&quot;&gt;q&lt;/span&gt; mínimo y &lt;span style=&quot; font-style:italic;&quot;&gt;q&lt;/span&gt; máximo.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignJustify|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="margin">
      <number>9</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout_9">
     <property name="margin">
      <number>9</number>
     </property>
     <item row="0" column="0">
      <widget class="QTabWidget" name="tabWidget">
       <property name="minimumSize">
        <size>
         <width>500</width>
         <height>0</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>&amp;Básicos</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="cgrlabel">
             <property name="text">
              <string>Secuencias:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTableView" name="sequenceTableView">
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::MultiSelection</enum>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
             <property name="iconSize">
              <size>
               <width>16</width>
               <height>16</height>
              </size>
             </property>
             <attribute name="horizontalHeaderMinimumSectionSize">
              <number>30</number>
             </attribute>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <attribute name="verticalHeaderDefaultSectionSize">
              <number>24</number>
             </attribute>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QGroupBox" name="qValuesGroupBox">
           <property name="title">
            <string>Valores para rango q:</string>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <layout class="QGridLayout" name="gridLayout_8">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_7">
              <item row="0" column="0">
               <widget class="QLabel" name="qMinLabel">
                <property name="text">
                 <string>q mínimo:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QSpinBox" name="minQSpinBox">
                <property name="minimumSize">
                 <size>
                  <width>80</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="minimum">
                 <number>-70</number>
                </property>
                <property name="maximum">
                 <number>-2</number>
                </property>
                <property name="value">
                 <number>-20</number>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QLabel" name="qMaxLabel">
                <property name="text">
                 <string>q máximo:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="3">
               <widget class="QSpinBox" name="maxQSpinBox">
                <property name="minimumSize">
                 <size>
                  <width>80</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="minimum">
                 <number>2</number>
                </property>
                <property name="maximum">
                 <number>70</number>
                </property>
                <property name="value">
                 <number>20</number>
                </property>
               </widget>
              </item>
              <item row="0" column="4">
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>&amp;Avanzados</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string>Radios</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="0" column="0">
               <layout class="QGridLayout" name="gridLayout_2">
                <item row="0" column="0">
                 <widget class="QLabel" name="MinRLabel">
                  <property name="minimumSize">
                   <size>
                    <width>125</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Radio mínimo (px):</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QSpinBox" name="minRSpinBox">
                  <property name="minimumSize">
                   <size>
                    <width>80</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="minimum">
                   <number>1</number>
                  </property>
                  <property name="maximum">
                   <number>250</number>
                  </property>
                 </widget>
                </item>
                <item row="0" column="2">
                 <widget class="QLabel" name="MaxRLabel">
                  <property name="frameShape">
                   <enum>QFrame::NoFrame</enum>
                  </property>
                  <property name="text">
                   <string>Radio máximo (px):</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="3">
                 <widget class="QSpinBox" name="maxRSpinBox">
                  <property name="minimumSize">
                   <size>
                    <width>80</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="minimum">
                   <number>1</number>
                  </property>
                  <property name="maximum">
                   <number>256</number>
                  </property>
                  <property name="value">
                   <number>256</number>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="radiusStepLabel">
                  <property name="minimumSize">
                   <size>
                    <width>150</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Salto entre radios (px):</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QSpinBox" name="radiusStepSpinBox">
                  <property name="minimumSize">
                   <size>
                    <width>80</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximum">
                   <number>32</number>
                  </property>
                  <property name="value">
                   <number>4</number>
                  </property>
                 </widget>
                </item>
                <item row="1" column="2">
                 <widget class="QLabel" name="totalRadiiLabel">
                  <property name="text">
                   <string>Total radios:</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="3">
                 <widget class="QLineEdit" name="totalRadii">
                  <property name="minimumSize">
                   <size>
                    <width>80</width>
                    <height>0</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item row="1" column="0">
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="0" column="0">
              <widget class="QLabel" name="nCentersLabel">
               <property name="text">
                <string>Número de centros:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QSpinBox" name="nCentersSpinBox">
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>0</height>
                </size>
               </property>
               <property name="minimum">
                <number>10</number>
               </property>
               <property name="maximum">
                <number>1000</number>
               </property>
               <property name="singleStep">
                <number>10</number>
               </property>
               <property name="value">
                <number>10</number>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="nIterationsLabel">
               <property name="text">
                <string>Número de iteraciones:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QSpinBox" name="nIterationsSpinBox">
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>0</height>
                </size>
               </property>
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>100000</number>
               </property>
               <property name="singleStep">
                <number>1</number>
               </property>
               <property name="value">
                <number>2</number>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="topMargin">
        <number>9</number>
       </property>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="buttonBox">
         <property name="locale">
          <locale language="Spanish" country="Colombia"/>
         </property>
         <property name="standardButtons">
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../resources/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MfaParametersForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>379</x>
     <y>313</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>314</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MfaParametersForm</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>433</x>
     <y>308</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
